llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -1654,6 +1654,12 @@ gentbl(
         strip_include_prefix = "lib/Target/" + target["name"],
         tbl_outs = target["tbl_outs"],
         tblgen = ":llvm-tblgen",
+        # MSVC isn't happy with long string literals, while other compilers
+        # which support them get significant compile time improvements with
+        # them enabled. Ideally this flag would only be enabled on Windows via
```

Comment:
Maybe make fixing this into a rule a TODO and file an issue to track? Can assign it to me if you want.

Reply:
Added a TODO linking to https://github.com/google/llvm-bazel/issues/92